nibabel/arrayproxy.py

```diff
@@ -162,6 +187,16 @@ def __getitem__(self, slicer):
         # Upcast as necessary for big slopes, intercepts
         return apply_read_scaling(raw_data, self._slope, self._inter)
 
+    def reshape(self, shape):
```

Comment:
Would be good to allow -1 as element in shape, for compatibility with numpy array reshape.